### PR TITLE
[embedding playground] Support index selection; Improve performance

### PR DESCRIPTION
**Pull request by shifucun**
image

- Make sure each model box renders only once. The encode and match time is substantial for larger model.
- Add checkbox for index selection
- Keep only one "apply" button

### CODE REVIEW

**Comment by shifucun** [comment]
@dwnoble We can ask UN to use this to testing their descriptions.

File: static/js/apps/eval_embeddings/app.tsx [hunk 1]
```diff
@@ -16,84 +16,89 @@
...
 import { EmbeddingObject } from "./util";
 
-const DESCRIPTION_TEXTAREA_ID = "description-textarea";
-
-const DEFAULT_DESCRIPTION =
-  'Count_Person,"population number"\ndc/topic/AgeMedians,"median age related"';
-const DEFAULT_QUERIES = "how many population\nmedian age";
+const DEFAULT_DESCRIPTION = 'Count_Person,"population number"';
+const DEFAULT_QUERY_STRING = "how many population";
+const DEFAULT_INDEX = "base_uae_mem";
```
Comment (pradh) [inline static/js/apps/eval_embeddings/app.tsx:27]: any way to highlight that this is prod by perhaps showing this as "base_uae_mem (prod)".  And perhaps as "medium_ft (custom-dc)"?

**Review by pradh — APPROVED** [review]
Very nice!

File: static/js/apps/eval_embeddings/app.tsx (hunk 1 above)
Comment (shifucun) [reply static/js/apps/eval_embeddings/app.tsx:27]: updated

**Comment by shifucun** [comment]
Thanks!

**Pull request merged**